### PR TITLE
solve a building bug on bazel version requirement

### PR DESCRIPTION
**Pull request by fayeshine**
@vrv 
today I also update bazel to newest master version, and when I rebuild tensorflow, I got this:

```
bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
ERROR: /home/wenjian/pkgs/tensorflow/WORKSPACE:21:1: Traceback (most recent call last):
        File "/home/wenjian/pkgs/tensorflow/WORKSPACE", line 21
                check_version("1.4")
        File "/home/wenjian/pkgs/tensorflow/tensorflow/tensorflow.bzl", line 6, in check_version
                fail("
Current Bazel version is {}, e...))

Current Bazel version is 0.2.0-2016-03-29 (@6b3294b), expected at least 1.4
.
ERROR: Error evaluating WORKSPACE file.
ERROR: no such package 'external': Package 'external' contains errors.
INFO: Elapsed time: 0.263s
```

So I found that the bazel's version number should be 0.x.x, not x.x
I changed `check_version("1.4")` to `check_version("0.14")` and the problem solved

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -18,7 +18,7 @@ tf_workspace()
 
 # Specify the minimum required bazel version.
 load("//tensorflow:tensorflow.bzl", "check_version")
-check_version("1.4")
+check_version("0.14")
```
Comment (vrv) [inline WORKSPACE]: this is probably also wrong.  I didn't know anyone was actually on bazel master, let me revert my original change :)

**Comment by vrv** [comment]
Rolled back in #1683 -- the version string was not what I expected it to be.  I'll roll it forward with the correct change soon.

**Comment by fayeshine** [comment]
@vrv , thanks

**Comment by vrv** [comment]
#1685 for the improved version that works

**Comment by fayeshine** [comment]
@vrv #1685  worked for me, thanks for acting quick to solve bugs. I'll remove my branch.

**Pull request closed without merging**